app/Http/Middleware/ApiVersion.php

```diff
@@ -0,0 +1,46 @@
+<?php
```

Comment:
Brad, there is a command named `composer cs-fix` that fixed the linting.  At this moment the file listing is failing, the following is a screenshot after running the command. Also please run `npm install` on you system so husky will catch the errors for you


![image](https://user-images.githubusercontent.com/41348080/95216559-d2ae9b00-07b7-11eb-8685-0adce8891910.png)